dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/informed-consent-template-verbal-covid.md

```diff
@@ -0,0 +1,73 @@
+While I and my team members have no symptoms and are not aware of any close contact with a COVID+ person, there is a risk of contamination or spread of COVID-19. We are minimizing the risk of this spread by following the most effective known mitigation strategies, as recommended by the **[Country]** health ministry and the World Health Organization:
+•	maintaining a distance of 2m/6ft
+•	wearing masks and gloves throughout
+•	providing you with a mask
+•	remaining outside  
+•	<any other protocols>
+We are also willing to follow any extra measure that would make you feel more comfortable when taking the survey.
+If in the next couple of weeks, we realize that we might have put you at risk (discovering for instance that a member of our team contracted the COVID-19 or has been in contact with a sick person), we will immediately contact you and refer you to the closest health facility to get tested. In the meantime, here / we will send you literature on COVID-19, its symptoms, how to limit transmission and what to do if you think you might have COVID-19.
+
+
+
+### Confidentiality
+Your study data will be handled as confidentially as possible.
+If results of this study are published or presented, individual names, and other personally identifiable information will not be used.
+No identifying information will be shared with any third party.
+We will ask for your consent in this study, and document it on this tablet.  
+
+To minimize the risks to confidentiality, we will keep all identifying information stored in encrypted files.
+Your responses will be numbered, and the code linking your name with your responses will also be stored in encrypted files on
+password-protected computers.
+Only trained interviewers and researchers from **[LIST OF ORGANIZATIONS]** will have access to these files.  
+
+When the research is completed, we may save the information for use in future research done by this research team.
+We will retain these records for up to **[NUMBER OF YEARS]** years after the study is over.
+The same measures described above will be taken to protect confidentiality of this study data.
+
+### Compensation
+You will not be paid for taking part in this study.
+
+### Rights
+Participation in research is completely voluntary.
+You are free to decline to take part in the project.
+You can decline to answer any questions and are free to stop taking part in the project at any time.
+Whether or not you choose to participate in the research and whether or not you choose to answer a question or continue
+participating in the project, there will be no penalty to you or loss of benefits to which you are otherwise entitled.
+
+### Questions
+If you have any questions about this research, you can ask us now or later at the details below.
+
+**[DETAILS OF CONTACT PERSON FROM SURVEY FIRM (INCLUDING NAME AND CONTACT NUMBER]**
+
+If you have any questions about your rights or treatment as a research participant in this study, you may contact:
```

Comment:
Do IRBs typically have a local contact number? i haven't actually ever seen this in practice

Reply:
Nope, but my understanding is we still have to provide the contact information (even if not local)